composer.json

```diff
@@ -68,7 +68,8 @@
         "installer-disable": true,
         "production-dependencies-guard": [
             "check-lock-file",
-            "check-abandoned"
+            "check-abandoned",
+            "white-list:woocommerce/subscriptions-core"
```

Comment:
To avoid this error during the process of `composer install`. I think that's OK given that `automattic/woocommerce-subscriptions-core` is in the archive mode, and we will soon phase it out from WooPayments.

```
Error: Dependencies guard has found violations in require-dependencies (source: lock-file):
 - woocommerce/subscriptions-core (via manifest): abandoned

In Guard.php line 91:
                                                                               
  Dependencies guard has found violations in require-dependencies (source: lo  
  ck-file):                                                                    
   - woocommerce/subscriptions-core (via manifest): abandoned            
```